### PR TITLE
Update UG: Troubleshooting, FAQ, Acknowledgements, and Glossary

### LINKED ISSUE #74: Editing User Guide: Troubleshooting, FAQ, Acknowledgements, Glossary
Glossary:
- Important to note down all of the terminologies that are used in our product and explain what each of them mean

### PR DESCRIPTION
**Pull request by dunliang0513**
## What
Update User Guide on the Troubleshooting, FAQ, Acknowledgements, and Glossary parts

## Why
Keep User Guide up-to-date

## How
NA

 Fix #74

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -118,8 +119,160 @@ Find applicants using keywords (name OR phone number OR both)
...
+### A
+> #### ADDRESS (Placeholder)
+> The ADDRESS is a text representing the address of the applicants.
+> #### Address
+> The address of the applicant provided by the user.
+> #### Admonitions
+> Admonition boxes (or simply “admonitions”) are coloured boxes containing highlighted pieces of text. <br>
+> #### Applicant
+> The person who apply to the company for the job.
+
```
Comment (FredericChow00) [inline docs/UserGuide.md:199]: can also add Application Cycle into the glossary

**Review by FredericChow00 — COMMENTED** [review]
Added one  possible term to Glossary

File: docs/UserGuide.md (hunk 1 above)
Comment (dunliang0513) [reply docs/UserGuide.md:199]: Ok noted~

**Review by bofeng1999 — COMMENTED** [review]
LGTM!

**Pull request merged**